app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from os import path
+import sys
+
+
+def copy_file(command: str) -> str or None:
+    cmd, main_file, new_file = command.split()
```

Comment:
This operation is potentially dangerous and may lead to unhandled exception due to inability to split the command string if its formatted incorrectly. You should `try:` this line for `ValueError` too if you wish to keep your code safe and robust